src/styles/getModalStyles.js

```diff
@@ -83,23 +83,19 @@ export default (type, windowDimensions) => {
         case CONST.MODAL.MODAL_TYPE.POPOVER:
             modalStyle = {
                 ...modalStyle,
+                ...anchorPosition,
                 ...{
+                    position: 'absolute',
                     alignItems: 'center',
                     justifyContent: 'flex-end',
-                    marginRight: windowWidth - variables.sideBarWidth,
-                    marginBottom: 100,
                 },
             };
             modalContainerStyle = {
-                width: variables.sideBarWidth - 40,
                 borderRadius: 12,
                 borderWidth: 1,
                 borderColor: themeColors.border,
-                paddingTop: 12,
-                paddingBottom: 12,
                 justifyContent: 'center',
                 overflow: 'hidden',
-                boxShadow: '0px 0px 10px 0px rgba(0, 0, 0, 0.025)',
```

Comment:
I'd prefer to keep these styles here. But I think the only way to make that work would be to create a `FAB_POPOVER` type and a `POPOVER` type of modal and then maybe share some styles between them. I think you'd end up with less code changes as well.

Reply:
I definitely disagree. These styles are very specific to the one popover we already had implemented, and modals of any type should be reusable.

Maybe we could leave the `boxShadow` here though.

Reply:
To clarify, by "here" I mean in this file. I agree modals should be reusable and making `POPOVER` more generic is fine, but I think the FAB popover styles should also live here for organizational purposes.